### ISSUE TITLE
Cubemaps is not working? [3.2.3]

### ISSUE THREAD

**Issue by QulixFedorov** [mistic100/Photo-Sphere-Viewer#174]
Single string for equirectangular panoramas is working, but I still can't view cubemaps variant. I tried to use example images from this project, but PSV show me nothing. But It loaded this 6 images.

Maybe I did something wrong?

```
panorama: {
  left:   'path/to/left.jpg',
  front:  'path/to/front.jpg',
  right:  'path/to/right.jpg',
  back:   'path/to/back.jpg',
  top:    'path/to/top.jpg',
  bottom: 'path/to/bottom.jpg'
}
```

Could you kindly help me with that? I don't saw any error at console.

**Comment by mistic100** [comment]
How could I help with so few info ?

Create an online demo of the issue.

**Comment by QulixFedorov** [comment]
Yeap, you right. Sorry for that. Let me show how I using that:

Require:
```
require([
               "../../components/photo-sphere-viewer/photo-sphere-viewer",
               "css!../../components/photo-sphere-viewer/photo-sphere-viewer",
              ], (photoSphereConstructor) => {
                       this.isPhotosphereViewerLoaded = true;
                       this.photoSphereConstructor = photoSphereConstructor;

                       this.addPanoramaInSphereViewer(this.activeMediaObject.original.url);

 });
```

Code: 
```
  public addPanoramaInSphereViewer(mediaUrl: string): void {

            var pano = [
                {
                    url: {
                        left: 'https://i.imgur.com/SMYTxzn.jpg',
                        front: 'https://i.imgur.com/1nXEpaF.jpg',
                        right: 'https://i.imgur.com/MEAXXWb.jpg',
                        back: 'https://i.imgur.com/WPdI4bi.jpg',
                        top: 'https://i.imgur.com/dh8hrGJ.jpg',
                        bottom: 'https://i.imgur.com/d6wvMJH.jpg'
                    },
                    desc: 'Cubemap'
                }
            ]

            this.photoSphereViewer = this.photoSphereConstructor({
                    container: 'panorama',
                    panorama: pano[0].url,
                    caption: pano[0].desc,
                    autoload: true,
                    time_anim: false
                });
}

yarn.lock:
```
photo-sphere-viewer@3.2.3:
  version "3.2.3"
  resolved "https://registry.yarnpkg.com/photo-sphere-viewer/-/photo-sphere-viewer-3.2.3.tgz#877b5777cfec78efbb025ba0d30c27378386e9e6"
  dependencies:
    d.js "~0.7.3"
    dot ">=1.0.3"
    three ">= 0.70.0"
    uevent "~1.0.0"
```

So, it works with single image, but when I tried use cubemap - it only loaded but show me nothing.
From console I got this: 

```
THREE.WebGLRenderer 89
-> three.min.js:905 THREE.MultiMaterial has been removed. Use an Array instead.
```

No errors. Nothing. 

So, I'm going to try your demo, but it not a solution for me. Maybe something wrong with dependencies?

**Comment by QulixFedorov** [comment]
Well. I removed `node_modules` folder, then run `yarn` and `npm` and load project. But no luck. 

PSV version is `3.2.3`.

About the demo. Demo is OK..

Any ideas what it can be? Why single image is working, but not cubemap?

**Comment by mistic100** [comment]
What do you mean "no errors" ?! You just copy-pasted an error.

This is already resolved for the upcoming version #165

**Comment by QulixFedorov** [comment]
Thank you for your time.
